spec/helpers/serializer_helper_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'spec_helper'
+
+describe SerializerHelper, type: :helper do
+  let(:serializer) do
+    class ExampleEnterpriseSerializer < ActiveModel::Serializer
+      attributes :id, :name
+    end
+    ExampleEnterpriseSerializer
+  end
```

Comment:
you could simply define and use ExampleEnterpriseSerializer without a let. There is no restriction on which scope you define a class.